microk8s-resources/actions/common/utils.sh

```diff
@@ -271,15 +271,7 @@ wait_for_service_shutdown() {
 }
 
 get_default_ip() {
```

Comment:
This is used in a couple of different places and cannot be changed so easily. For example `microk8s.config` will not be returning the kubeconfig to access microk8s in a VM.

Reply:
@ktsakalozos: It looks like with this PR, there's two opposing use cases, making microk8s available from outside a VM, and making microk8s available at a magic IP address (which wouldn't be available from outside a VM). Should we introduce a flag such as `--magic/--no-magic` to `microk8s.config` that would disambiguate between these two use cases?

Reply:
`microk8s.config` will give you a kubeconfig. If you are sitting in front of the host the kubeconfig would better have the 127.0.0.1 endpoint for accessing the apiserver. If MicroK8s is in a VM and/or you are not on the host you should be using the endpoint though which you can access the apiserver and this is probably the default IP and not the magic IP. So I am not sure when you would want a kubeconfig with the magic IP on.

The usecase I see for the magic IP is when you have a service on the host and you need to access it from a pod. I this case the default IP is not good because it may change and you may not want to make that service publicely available.